packages/core/src/codewhisperer/service/recommendationHandler.ts

```diff
@@ -62,6 +62,8 @@ const nextCommand = Commands.declare('editor.action.inlineSuggest.showNext', ()
 
 const rejectCommand = Commands.declare('aws.codeWhisperer.rejectCodeSuggestion', () => async () => {
     RecommendationHandler.instance.reportUserDecisions(-1)
+
+    await Commands.tryExecute('aws.codewhisperer.refreshAnnotation')
```

Comment:
I was trying
both
(1.) to use declared and exported command `aws.codewhisperer.refreshAnnotation`
and 
(2.)
```
Container.instance.lineAnnotationController.refresh()
```

however both encountered "circular dependency" issue, thus having to set back to this approach which may not be preferred